src/queries/a11yState.ts

```diff
@@ -11,14 +12,20 @@ import type {
   QueryAllByQuery,
   QueryByQuery,
 } from './makeQueries';
+import { AccessibilityOption } from './accessibilityOption';
 
 const queryAllByA11yState = (
   instance: ReactTestInstance
-): ((matcher: AccessibilityState) => Array<ReactTestInstance>) =>
-  function queryAllByA11yStateFn(matcher) {
-    return instance.findAll(
+): ((
+  matcher: AccessibilityState,
+  queryOptions?: AccessibilityOption
+) => Array<ReactTestInstance>) =>
+  function queryAllByA11yStateFn(matcher, queryOptions) {
+    return findAll(
+      instance,
       (node) =>
-        typeof node.type === 'string' && matchAccessibilityState(node, matcher)
+        typeof node.type === 'string' && matchAccessibilityState(node, matcher),
+      { hidden: queryOptions?.hidden }
```

Comment:
```suggestion
      queryOptions
```

Reply:
i wondered about that actually, that's what i did at first and then thought that it did not necessarily make sense to pass all options to findAll when all he needs is hidden, wdyt ?

Reply:
From typing perspective, passing whole `options` object seems fine and natural, as it implement the relevant interface, hence we will should only access the props defined in that interface.